docs/pages/toolchain/sozo/common-options/configurations.md

```diff
@@ -0,0 +1,47 @@
+# Account Options
+
+This module provides options for configuring an account in the `sozo` command.
+
+## Configuration Priorities
+
+The configuration system follows the following priority order:
+
+1. Command-line arguments: If options are specified via command-line arguments, they have the highest priority.
+2. Environment variables: Environment variables provide a way to configure options globally.
+3. Configuration in `Scarb.toml`: Any configuration found within the `Scarb.toml` file will be considered if options are not found in the first two steps.
+
+## Account Options
+
+- `account_address`: StarkNet account address. Can be specified via command-line arguments or environment variables.
+- `signer`: Options related to signer configuration.
+- `legacy`: Enables the use of a legacy account (cairo0 account).
+
+## Public Methods
+
+- `account(provider, env_metadata)`: Creates a StarkNet account using the specified options. Takes into account the configuration priorities to determine the account address and other settings.
+- `account_address(env_metadata)`: Returns the StarkNet account address. Takes into account the configuration priorities.
```

Comment:
Aren't those inside the sozo code? Not sure if a sozo user should know about this technical specificity. Or what was the idea behind this?